src/Token.php

```diff
@@ -65,21 +65,21 @@ public static function getTokenKindNameFromValue($kindName) {
     function jsonSerialize() {
         $kindName = $this->getTokenKindNameFromValue($this->kind);
 
-        if (!isset($GLOBALS["SHORT_TOKEN_SERIALIZE"])) {
-            $GLOBALS["SHORT_TOKEN_SERIALIZE"] = false;
+        if (!isset($GLOBALS['SHORT_TOKEN_SERIALIZE'])) {
```

Comment:
As far as I'm aware, there is no particular convention on quoting style. Don't think it makes sense to change here, especially as other parts of the project predominantly use double quotes.